[fix #11] [mozmeao/infra#1104] Clean up oregon-a admin-stage yaml.

jpetto opened the pull request.

jpetto committed:
[fix #11] [mozmeao/infra#1104] Clean up oregon-a admin-stage yaml.

jgmize commented on oregon-a/basket-admin-stage/web-deploy.yaml:
```diff
@@ -48,261 +37,155 @@ spec:
-            secretKeyRef:
-              key: redis-cache-db
-              name: basket-admin-stage-v89-env
+          value: 3
```
```suggestion
          value: "3"
```

jgmize commented on oregon-a/basket-admin-stage/web-deploy.yaml:
```diff
@@ -48,261 +37,155 @@ spec:
-            secretKeyRef:
-              key: redis-celery-db
-              name: basket-admin-stage-v89-env
+          value: 2
```
```suggestion
          value: "2"
```

jgmize commented on oregon-a/basket-admin-stage/web-deploy.yaml:
```diff
@@ -48,261 +37,155 @@ spec:
-              key: wsgi-num-workers
-              name: basket-admin-stage-v89-env
-        image: 127.0.0.1:5555/basket-admin-stage:v89
+          value: 4
```
```suggestion
          value: "4"
```

jgmize commented on oregon-a/basket-admin-stage/web-deploy.yaml:
```diff
@@ -48,261 +37,155 @@ spec:
-              name: basket-admin-stage-v89-env
-        image: 127.0.0.1:5555/basket-admin-stage:v89
+          value: 4
+        image: mozmeao/basket:4d0668d
```
```suggestion
        image: mozmeao/basket:4d0668d4e5cb304e68b04bcc0d2b278d27b591e6
```

jpetto committed:
Review fixes:

- remove superfluous data from ns.yaml
- quote all numeric env var values
- use full image hash
